### DISCUSSION TITLE
How to disable user interactions?

### DISCUSSION THREAD

**Discussion by ilslv9** [sceneview/sceneview#506] [Q&A] [answered]
Hi,
I'm working on a project which uses SceneView. We want to find a way how to block user interactions with the view to have a fixed projection of 3D model. Is there any API or preferable way to do it? Unfortunately, we didn't find a way in API to do so and as a workaround we use ```view.setOnTouchListener { _, _ -> true }```

**Comment by ridcore** [comment, marked as answer]
By default, when you position a modelNode in the scene, it has the isEditable and isTouchable properties set to true, it would be enough to set them to false when creating the model

[ModelNode Class](https://sceneview.github.io/api/sceneview-android/sceneview/io.github.sceneview.node/-model-node/index.html?query=open%20class%20ModelNode(val%20modelInstance:%20ModelInstance,%20autoAnimate:%20Boolean%20=%20true,%20scaleToUnits:%20Float?%20=%20null,%20centerOrigin:%20Position?%20=%20null)%20:%20Node)
